src/types.ts

```diff
@@ -116,6 +116,7 @@ export interface Question {
   label: string;
   type: string;
   questionOptions: QuestionOptions;
+  datePickerFormat?: string;
```

Comment:
This property got renamed to `datePickerFormat` in the React form engine [here](https://github.com/openmrs/openmrs-form-engine-lib/pull/312).